{"levelname": "ERROR", "asctime": "2023-11-02T19:43:34.002Z", "name": "processing", "message": "Unable to publish collection version.", "lineno": 69, "pathname": "/single-cell-data-portal/backend/layers/processing/publish_revisions.py", "collection_id": "bc758887-4c68-4159-8b6b-c0d99d7e2cbd", "collection_version_id": "23595cd6-d15c-482b-9742-c28e9ef67320", "errors": ["error message"]}
